bukuserver/templates/bukuserver/statistic.html

```diff
@@ -223,29 +246,35 @@ <h4 class="modal-title" id="myModalLabel">Title ranking</h4>
 
 {% block tail %}
   {{ super() }}
+  <script>// sticky table headers in modals
+    $('.modal-body').each(function () {
+      $('thead', this).css({top: '-' + $.css(this, 'padding-top')});
+    });
+  </script>
```

Comment:
Sticky header `top:` must equal the (negative) distance to the top of the scrollbox